### DISCUSSION TITLE
Finding this on the marketplace?

### DISCUSSION THREAD

**Discussion by Fentonator** [microdee/OpenUnrealSpaceMouse#20] [Q&A] [answered]
Very excited to try this out, I've been wanting support for my SMPro in UE4 for literally years now, after finding what a profound difference it made in Maya. I noticed that the main page indicates that it is available in the Marketplace, but I haven't had any luck finding it using obvious search terms -- "3d mouse", "space mouse", "spacemouse", etc.

If anyone could point me at a way to actually find it, it would be much appreciated. I can certainly build it myself if it comes down to that, but I'd also like to share it with a number of other folks I'm working with on a project, and that would be a lot easier through a marketplace plugin.

**Comment by microdee** [comment, marked as answer]
Oh the marketplace thing is under "Would be nice to have", it's not there yet, I still need to do some refactoring to be compatible with the Marketplace requirements, but that will probably only happen during christmas/new-year.
You can also just include the release for appropriate engine version in the project plugins if you want to share it with team-mates.

**Reply by Fentonator** [reply]
Whoops, apparently I just can't actually read. Well, just consider it a vote for putting it up there when you get around to it, then. In the meanwhile I'll poke at getting it wrangled in by hand.

**Comment by microdee** [comment]
I've finally put together the store page and updated the code to comply with Marketplace guidelines and now it's awaiting approval. I hope they'll accept it and then you can finally install this plugin from the launcher

**Reply by Fentonator** [reply]
Excellent, many thanks -- I'll keep an eye out for it.
